panel/browser-panel.hpp

```diff
@@ -67,13 +68,19 @@ struct QCef {
 					 QObject *obj) = 0;
 };
 
+static void *get_browser_lib()
+{
+	obs_module_t *browserModule = obs_get_module_by_name("obs-browser");
```

Comment:
I just tried to port this over to the mac branch and got a "use of undeclared identifier": there apparently is no such function. Is this a typo?

Reply:
Ah, https://github.com/obsproject/obs-studio/pull/3871.